arex-storage-web-api/src/main/java/com/arextest/storage/service/AgentWorkingService.java

```diff
@@ -57,16 +57,40 @@ public class AgentWorkingService {
    * @return true means success,else save failure
    */
   public <T extends Mocker> boolean saveRecord(@NotNull T item) {
+    if (!prepareAndDispatchRecord(item)) {
+      return false;
+    }
+
+    RepositoryProvider<T> repositoryWriter = repositoryProviderFactory.defaultProvider();
+    return repositoryWriter != null && repositoryWriter.save(item);
+  }
+
+  public <T extends Mocker> boolean batchSaveRecord(@NotNull List<T> list) {
+    if (CollectionUtils.isEmpty(list)) {
+      return false;
+    }
+
+    for (T item : list) {
+      if (!prepareAndDispatchRecord(item)) {
```

Comment:
Is this method mainly used to intercept some operations when saving mockers？like: sql parse?